hap-ibd/dockerfile

```diff
@@ -0,0 +1,50 @@
+################## BASE IMAGE ######################
+FROM adoptopenjdk:8-jre@sha256:bf736024c65942ab045c497e408706f9e599651bf8c01a75ecb3d6971d66700d
+
+################## METADATA ######################
+LABEL base_image="biocontainers:vdebian-buster-backports_cv1"
+LABEL version="1"
+LABEL software="hap-ibd"
+LABEL software.version="1.0, 20May22.818"
```

Comment:
Incorrect versiin labelling (comma , spaces,...) and what is this version??

Reply:
I took the version directly from the tool, as requested by the author:https://github.com/browning-lab/hap-ibd

https://github.com/browning-lab/hap-ibd/commit/46b9a14f21ef8a5f86352889caadb0a6cc2a9fe0